packages/portalnetwork/test/integration/wire.spec.ts

```diff
@@ -90,6 +88,7 @@ tape('Portal Wire Spec Testing', async (t) => {
     const child = spawn(process.execPath, [file])
     let portal1: PortalNetwork
     let portal2: PortalNetwork
+    tape.onFailure(() => end(child, [portal1, portal2], st))
```

Comment:
@ryanio I discovered this `tape.onFailure()` method while triaging a hanging integration test for this PR.  When you combine it with a `timeout` in an async test, it makes for a nice way to clean-up things that cause tests to hang so they still terminate if there's a failure.  Could be a helpful additional tool in cleaning up tests in the `client ` if we identify new race conditions/tests that occasionally fail due to open handles.

Reply:
cool, thanks for the tip!